Models/repo.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+(function(module) {
+  const repos = {};
+
+  repos.all = [];
+
+  repos.requestRepos = function(callback) {
```

Comment:
Where are you calling this? (Hint: it should be in your route that you need to create for your route nav link!)